packages/fxa-settings/src/components/FlowContainer/index.tsx

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { RouteComponentProps, useLocation, Link } from '@reach/router';
+import { ReactComponent as BackArrow } from './back-arrow.svg';
+
+type FlowContainerProps = {
+  title?: string;
+  backRoute?: string;
+};
+
+export const FlowContainer = ({
+  title,
+  backRoute,
+}: FlowContainerProps & RouteComponentProps) => {
+  const location = useLocation();
+  const backRoutePath =
+    (backRoute ? backRoute : '/beta/settings') + location.search;
+  return (
+    <div className="flex" data-testid="flow-container">
+      <Link
+        to={backRoutePath}
+        className="pt-7 pr-6"
+        data-testid="flow-container-back-btn"
+      >
+        <BackArrow className="inline h-8 w-8" />
```

Comment:
This could use a `role="img"`. Also an `aria-label="alt text"`  _or_ the link can have a `title="alt text"` if it should have hover text. Might check with Meridel W. on what copy should be read and if this should have a title.